lib/validate_test.go

```diff
@@ -81,3 +82,67 @@ func TestValidateSurveyLink(t *testing.T) {
 	}
 
 }
+
+// TestValidateQuotaPlan testing quota plan validation
+func TestValidateQuotaPlan(t *testing.T) {
+	tables := []struct {
+		name     string
+		input    string
+		expected error
+	}{
+		{
+
+			"Case 1: Happy path, everything is valid with full quota plan",
```

Comment:
Can we add a test case for when the both/ one of filters and quota groups are sent in as empty arrays?